### DISCUSSION TITLE
How to add swatches on Crosssell Products on Cart page?

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#36] [1. Help]
Hi there,

As I am working with Breeze evolution on my website, The color and size swatches are showing on the Category [PLP] page. But I want to show the same on the Crosssell products slider on the Cart page.

So, Ho can I add or implement this? Please guide me on the same.

Thanks

**Comment by vovayatsyuk** [comment]
Sorry, I don't know how to do this. Try to use the same approach you would use for the Luma theme. Maybe there are some third-party modules that can do this.

**Reply by rutulritz** [reply]
But, on PLP it is rendering from your Breeze module.

**Reply by vovayatsyuk** [reply]
Breeze is using Magento templates and xml layout updates. The only thing we change -is custom js implementation. So, if you’ll add swatches on luma, you can do the exactly same solution on breeze.

**Reply by rutulritz** [reply]
Ok I will try that
